server/protocols/webadmin/webadmin-mailrepository/src/main/java/org/apache/james/webadmin/service/ReprocessingService.java

```diff
@@ -75,6 +91,14 @@ public Optional<String> getTargetProcessor() {
         public boolean isConsume() {
             return consume;
         }
+
+        public int getRateLimit() {
+            return rateLimit;
+        }
+
+        public Duration getRateLimitUnit() {
+            return rateLimitUnit;
+        }
```

Comment:
```suggestion
        public Limit getLimit() {
            return limit;
        }
```

(We already should have a nice POJO objects for limit